cmd/kubeadm/app/phases/addons/manifests.go

```diff
@@ -64,6 +64,14 @@ spec:
       labels:
         k8s-app: kube-proxy
     spec:
+      initContainers:
+      - name: touch-lock
+        image: {{ .ImageRepository }}/busybox-{{ .Arch }}
```

Comment:
@cmluciano How critical is this? Can we afford not having this in v1.7?
Where is the related bug/issue?